bindings/cpp/src/lib.rs

```diff
@@ -26,6 +27,12 @@ mod ffi {
         value: String,
     }
 
+    enum SeekDir {
```

Comment:
How about using `SeekFrom`, seems we still need a converation bettwen ccp `seekdir` to rust `SeekFrom`. And `SeekDir` has different meaning in rust.

Reply:
We can import `libc` crate to avoid exposing c++ std struct by ourselves.

Reply:
already rename `SeekDir` to `SeekFrom` in rust.